release/rotation

```diff
@@ -2,6 +2,7 @@
 # to access an ad-hoc machine for Windows testing. This can be done by opening a PR
 # like the following: https://github.com/DACH-NY/daml-language-ad-hoc/pull/37
 U0PNNPY05 robin-da
+U03518HUVBP stewSquared
```

Comment:
confirmed :+1:

Reply:
This means you'll be in charge of the 2.1 snapshot release on Wed 2022-03-09.

Reply:
Noted! Happy to learn how it works